assets/css/main.css

```diff
@@ -115,6 +115,48 @@ body {
   background: #fff !important;
 }
 
+.uppercase {
+  text-transform: uppercase !important;
+}
+.textCenter {
+  text-align: center !important;
```

Comment:
css classes should be hyphen-delimited rather than camelCase (ie. text-center instead of textCenter)